.github/workflows/ci.yml

```diff
@@ -8,6 +8,7 @@ on:
       - master
       - stable
       - v*
+      - support_pgc
```

Comment:
Fine for a WIP, but just to let you know, you can also trigger a build in the GitHub GUI by going to Actions -> CI -> Trigger Workflow -> pick branch (`support_pgc`).

Reply:
Hi @henryiii , 

OK. I was trying to find it but w/o any result.

Thanks in any case!